src/siteConstants.js

```diff
@@ -17,3 +17,14 @@ export const YOUTUBE = 'https://www.youtube.com/c/IjemmaOnwuzulike';
 // Projects
 export const NKOWAOKWU = 'https://nkowaokwu.com';
 export const NKOWAOKWU_CHROME = 'https://nkowaokwu.com/chrome';
+
+// Auth
+export const JWT_SECRET = '@developer@NkowaOkwu@secret@key@';
```

Comment:
we wouldn't want to commit a JWS_SECRET since this an open-source project and it opens us up to security vulnerabilities - rather we would want to save this JWT token in our GitHub actions (a secure location)
```suggestion
export const JWT_SECRET = process.env.JWT_SECRET;
```

I can update the environment variable after you merge this PR so production code will be able to use the JWT_SECRET stored in GitHub